e2e_tests/tests/deploy/test_local.py

```diff
@@ -114,6 +114,24 @@ def test_cluster_down() -> None:
     assert len(containers) == 0
 
 
+@pytest.mark.det_deploy_local
+def test_ee_cluster_up() -> None:
+    name = "test_ee_cluster_up"
+
+    with resource_manager(Resource.CLUSTER, name, {}, ["no-gpu", "enterprise-edition"]):
+        container_name = name + "_determined-master_1"
+        client = docker.from_env()
+
+        containers = client.containers.list(filters={"name": container_name})
+        assert len(containers) > 0
+
+        for container in containers:
+            if "Config" in container.attrs and "Image" in container.attrs["Config"]:
```

Comment:
How many times do you expect this predicate to be true? Is this expectation worth validating?

If those expectations aren't clear in the code (assuming you add validation), then a short comment would be helpful.

Reply:
I expect this to be true most of the time - I read the code comments for docker containers and this function is equivalent to `docker ps`, which should show the image name consistently. I'll add a comment just to make it clear